libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/security/Encryptor.java

```diff
@@ -422,7 +439,11 @@ public static String decryptWithRSA(PrivateKey privateKey, String data) {
      * @return Decrypted data.
      */
     public static byte[] decryptWithRSABytes(PrivateKey privateKey, String data) {
-        return decryptWithPrivateKey(privateKey, data, RSA_PKCS1);
+        byte[] result =  decryptWithPrivateKey(privateKey, data, RSA_ECB_OAEP, /* logErrorOnFailure */ false);
```

Comment:
App upgrading to 11.1.1 will fail to decrypt keys with OAEP and it's okay, they should succed to decrypt them with RSA_PKCS1. Only logging a warning if OAEP fails. An error will be logged if we completely failed to decrypt the data.